pkg/services/rpcsrv/server.go

```diff
@@ -1719,9 +1715,6 @@ func (s *Server) getStateRoot(ps params.Params) (any, *neorpc.Error) {
 
 func (s *Server) getStorage(ps params.Params) (any, *neorpc.Error) {
 	id, rErr := s.contractIDFromParam(ps.Value(0))
-	if rErr == neorpc.ErrUnknown {
-		return nil, nil
-	}
```

Comment:
This one is a separate behaviour-changing patch, please, create a separate commit for it. And actually, it's a bug. Take a look at the reference code, current behaviour matches your patch: https://github.com/neo-project/neo-modules/blob/99ffc846e4df93491d1d812cb4c6ef6ee875e056/src/RpcServer/RpcServer.Blockchain.cs#L187. These lines were changed way back in https://github.com/neo-project/neo-modules/commit/237ef7d057265723571efde74e54001f68ce969d, and we've skipped this change. Add a note about it in the commit message.

Reply:
Tnx, added a separate commit